library/tinkerpop/src/test/java/uk/gov/gchq/gaffer/tinkerpop/GafferPopGraphTest.java

```diff
@@ -224,19 +396,122 @@ public void shouldAddAndGetEdge() {
         // Then
         final Edge edge = edges.next();
         assertFalse(edges.hasNext()); // there is only 1 vertex
-        assertEquals(VERTEX_1, ((List) edge.id()).get(0));
-        assertEquals(VERTEX_2, ((List) edge.id()).get(1));
-        assertEquals(CREATED_EDGE_GROUP, edge.label());
-        assertEquals(gafferPopInVertex, edge.inVertex());
-        assertEquals(gafferPopOutVertex, edge.outVertex());
-        assertEquals(1.5, (Double) edge.property(WEIGHT_PROPERTY).value(), 0);
+        assertThat(((List) edge.id()).get(0)).isEqualTo(VERTEX_1);
+        assertThat(((List) edge.id()).get(1)).isEqualTo(VERTEX_2);
+        assertThat(edge.label()).isEqualTo(CREATED_EDGE_GROUP);
+        assertThat(edge.inVertex()).isEqualTo(gafferPopInVertex);
+        assertThat(edge.outVertex()).isEqualTo(gafferPopOutVertex);
+        assertThat((Double) edge.property(WEIGHT_PROPERTY).value()).isEqualTo(1.5);
```

Comment:
No longer requires a cast:
```suggestion
        assertThat(edge.property(WEIGHT_PROPERTY).value()).isEqualTo(1.5);
```